src/components/Profile.tsx

```diff
@@ -48,7 +53,7 @@ const Profile = (): JSX.Element => {
           />
           <div className="Profile__personalInfo">
             <label className="Profile__personalInfoLabel">NAME</label>
-            <div className="Profile__name">{profileName}</div>
+            <div className="Profile__name">{profileName || userId}</div>
```

Comment:
I think this would be a little cleaner if you handled the fallback value when you defined the variable. Also "Annoymous" or some such might be a better option than the userId here.

Reply:
As long as it's spelled "Anonymous"